### PR TITLE
update demarks pre 0.13

### PR DESCRIPTION
**Pull request by alanlujan91**

### CODE REVIEW

**Comment by alanlujan91** [comment]
@llorracc this is not running automated tests

**Comment by llorracc** [comment]
@alanlujan91, what does this PR do?

**Comment by alanlujan91** [comment]
Right now, Demarks do not work with master branch of HARK. This should bring them up to date. 

Also, I found that the automatic tests for Demarks hadn't run in a few months, so who knows for how long Demarks have been out of sync.

**Comment by llorracc** [comment]
Took a quick look, but it was hard to find significant changes that should be reviewed among the mass of junk changes (to version numbers, spacing, etc).  If it passes the automated tests, I'm OK with merging.

**Comment by alanlujan91** [comment]
Part of this PR is syncing and executing all the notebooks/jupytext files to see any changes that have happened with master branch of HARK. Presumably changes should be none/small, but we won't know unless we execute and upload changes to the notebooks.

File: notebooks/Micro-and-Macro-Implications-of-Very-Impatient-HHs.py [hunk 1]
```diff
@@ -237,7 +256,7 @@ def in_ipynb():
 DiscFac_spread = 0.0085  # spread of beta distribution
 DiscFac_dstn = (
     Uniform(DiscFac_mean - DiscFac_spread, DiscFac_mean + DiscFac_spread)
-    .approx(num_types)
+    .discretize(num_types)
```
Comment (alanlujan91) [inline notebooks/Micro-and-Macro-Implications-of-Very-Impatient-HHs.py:259]: @llorracc indeed @dedwar65's issue is corrected in this PR

**Comment by dedwar65** [comment]
Thanks, I was able to fetch this PR and then make the appropriate changes in my own notebook.

Will we still meet? Any time this evening should work for me. I am still having the issue of opening the cstwMPC notebook.

Decory

From: alanlujan91 ***@***.***>
Date: Monday, January 23, 2023 at 4:32 PM
To: econ-ark/DemARK ***@***.***>
Cc: Decory Edwards ***@***.***>, Mention ***@***.***>
Subject: Re: [econ-ark/DemARK] update demarks pre 0.13 (PR #190)

      External Email - Use Caution





@alanlujan91 commented on this pull request.

________________________________

In notebooks/Micro-and-Macro-Implications-of-Very-Impatient-HHs.py<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fecon-ark%2FDemARK%2Fpull%2F190%23discussion_r1084582719&data=05%7C01%7Cdedwar65%40jhu.edu%7C051938f8685d45a2487e08dafd895f00%7C9fa4f438b1e6473b803f86f8aedf0dec%7C0%7C0%7C638101063702848541%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=J5v12w%2Bj0p9UmFviPFUAbczu6WthpVLBEKOd5Y563tg%3D&reserved=0>:

> @@ -237,7 +256,7 @@ def in_ipynb():

 DiscFac_spread = 0.0085  # spread of beta distribution

 DiscFac_dstn = (

     Uniform(DiscFac_mean - DiscFac_spread, DiscFac_mean + DiscFac_spread)

-    .approx(num_types)

+    .discretize(num_types)

@llorracc<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fllorracc&data=05%7C01%7Cdedwar65%40jhu.edu%7C051938f8685d45a2487e08dafd895f00%7C9fa4f438b1e6473b803f86f8aedf0dec%7C0%7C0%7C638101063702848541%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=D6rof4p4NaBBYssLyMzuvCA0uQRrkOesNNzdshnb51k%3D&reserved=0> indeed @dedwar65<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdedwar65&data=05%7C01%7Cdedwar65%40jhu.edu%7C051938f8685d45a2487e08dafd895f00%7C9fa4f438b1e6473b803f86f8aedf0dec%7C0%7C0%7C638101063702848541%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=NWTGXvPpwfCBE7iG0AWUjICF8%2BPKSFcfNIwKqi8M%2B%2Bc%3D&reserved=0>'s issue is corrected in this PR

**Comment by llorracc** [comment]
Merging after @dedwar65 pointed out that without the merge some DemARKs break.

**Pull request merged**